lib/kredis/types.rb

```diff
@@ -5,37 +5,36 @@ def proxy(key, config: :shared, after_change: nil)
     type_from(Proxy, config, key, after_change: after_change)
   end
 
-
-  def scalar(key, typed: :string, default: nil, config: :shared, after_change: nil)
-    type_from(Scalar, config, key, after_change: after_change, typed: typed, default: default)
+  def scalar(key, typed: :string, default: nil, config: :shared, after_change: nil, expires_in: nil)
```

Comment:
Restore another newline that was removed in the merge conflict resolution.

```suggestion

    def scalar(key, typed: :string, default: nil, config: :shared, after_change: nil, expires_in: nil)
```